src/Sylius/Bundle/AdminBundle/gulpfile.babel.js

```diff
@@ -0,0 +1,167 @@
+            vendorUiPath + 'Resources/private/js/**',
+            vendorAdminPath + 'Resources/private/js/**',
+        ],
+        sass: [
+            vendorUiPath + 'Resources/private/sass/**',
+            vendorAdminPath + 'Resources/private/sass/**',
+        ],
+        css: [
+            nodeModulesPath + 'semantic-ui-css/semantic.min.css',
+            vendorUiPath + 'Resources/private/css/**',
+            vendorAdminPath + 'Resources/private/css/**',
+        ],
+        img: [
+            vendorUiPath + 'Resources/private/img/**',
+            vendorAdminPath + 'Resources/private/img/**',
+        ],
+    },
+};
+
+const sourcePathMap = [
+    {
+        sourceDir: path.relative('', vendorAdminPath + 'Resources/private/'),
+        destPath: '/SyliusAdminBundle/',
+    },
+    {
+        sourceDir: path.relative('', vendorUiPath + 'Resources/private/'),
+        destPath: '/SyliusUiBundle/',
+    },
+    {
+        sourceDir: path.relative('', nodeModulesPath),
+        destPath: '/node_modules/',
+    },
+];
+
+const mapSourcePath = function mapSourcePath(sourcePath, file) {
+    for (const { sourceDir, destPath } of sourcePathMap) {
+        if (sourcePath.substring(0, sourceDir.length) === sourceDir) {
+            return destPath + sourcePath.substring(sourceDir.length);
+        }
+    }
+
+    return sourcePath;
+};
+
+export const buildAdminJs = function buildAdminJs() {
+    return gulp.src(paths.admin.js, { base: './' })
+        .pipe(gulpif(env !== 'prod', sourcemaps.init()))
+        .pipe(concat('app.js'))
+        .pipe(gulpif(env === 'prod', uglify()))
+        .pipe(gulpif(env !== 'prod', sourcemaps.mapSources(mapSourcePath)))
+        .pipe(gulpif(env !== 'prod', sourcemaps.write('./')))
+        .pipe(gulp.dest(adminRootPath + 'js/'))
+        .pipe(livereload())
+    ;
+};
+buildAdminJs.description = 'Build admin js assets.';
+
+export const buildAdminCss = function buildAdminCss() {
+    const copyStream = merge(
+        gulp.src(nodeModulesPath+'semantic-ui-css/themes/**/*')
+            .pipe(gulp.dest(adminRootPath + 'css/themes/'))
+        ,
+    );
+
+    const cssStream = gulp.src(paths.admin.css, { base: './' })
+        .pipe(gulpif(env !== 'prod', sourcemaps.init()))
+        .pipe(concat('css-files.css'))
+    ;
+
+    const sassStream = gulp.src(paths.admin.sass, { base: './' })
+        .pipe(gulpif(env !== 'prod', sourcemaps.init()))
+        .pipe(sass())
+        .pipe(concat('sass-files.scss'))
+    ;
+
+    return merge(
+        copyStream,
+        merge(cssStream, sassStream)
+            .pipe(order(['css-files.css', 'sass-files.scss']))
+            .pipe(concat('style.css'))
+            .pipe(gulpif(env === 'prod', uglifycss()))
+            .pipe(gulpif(env !== 'prod', sourcemaps.mapSources(mapSourcePath)))
+            .pipe(gulpif(env !== 'prod', sourcemaps.write('./')))
+            .pipe(gulp.dest(adminRootPath + 'css/'))
+            .pipe(livereload())
+        ,
```

Comment:
I don't like this comma too... But I'm not sure where else to put it. :see_no_evil: